lighthouse-core/computed/page-dependency-graph.js

```diff
@@ -137,6 +136,23 @@ class PageDependencyGraph {
         children.push(traceOfTab.mainThreadEvents[i]);
       }
 
+      // Skip trace events that do not have a sizable duration,
+      // unless the event is the first ParseHTML/Layout/Paint.
```

Comment:
add a link to my explanatory comment on that issue?

Reply:
Yes sir Help Me grow